docs/configuration.md

```diff
@@ -36,17 +49,17 @@ providers:
 
 #### Authentication as a GitHub Account
 
-lookout needs to authenticate with GitHub. The easiest method for testing purposes is using a [GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/). The token should have the **repo** scope enabled.
+**lookout** needs to authenticate with GitHub. The easiest method for testing purposes is using a [GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/). The token should have the `repo` scope enabled.
 
-The credential can then be passed to `lookoutd` as:
+The credential can then be also passed to `lookoutd` as:
```

Comment:
Why did you add `also`? In addition to what?

Reply:
I see. But if you read this sentence, it says "also" when there is no previous alternative. This is the first one, it sounds weird to say "also".

Reply:
I think now is more clear: 6ff7b2411587c8f551326c3d30107be0b6573315
Any other idea @carlosms